aws/aws-explanation/ec2-image-retention-policy.rst

```diff
@@ -3,7 +3,7 @@ AWS image retention policies
 
 Ubuntu images on AWS have three stage of life cycle : *release* > *deprecation* > *deletion*. 
 
-Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be launched from the `AWS CLI`_ using its AMI ID. You can also find and view details of deprecated images using the AWS CLI `describe-images`_ command by including the ``--include-deprecated`` flag.
+Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be launched from the `AWS CLI`_ using its AMI ID. You can also find and view details of deprecated images using the AWS CLI `describe-images`_ command by including the ``--include-deprecated`` flag. Unlike deprecated images, private images are not available to use and invisible to describe-images. This step of marking an image private is reserved for AMIs that go unused and have been long since replaced by newer images.
 This policy determines when an image will be deleted. Deleted images are no longer accessible for use, but instances already launched with those images will not be affected.
```

Comment:
Maybe we should move this statement into the beginning of the next section (Image retention policy).